app/views/dashboard/index.html.erb

```diff
@@ -1,9 +1,3 @@
-<form role="search">
-  <div class="form-group">
-    <input type="text" id="search" class="form-control" placeholder="Search reasons">
-  </div>
-</form>
-
```

Comment:
This should still be functional / not dead?

Reply:
No, this never has worked... It just displays a search box but it does not actually do anything (i.e. won't function at all)

Reply:
I've heard that complaint from others, but it seems to work on prod on my machine so I'd like to leave it.